src/addDelegator.ts

```diff
@@ -59,7 +59,7 @@ export async function getUnsignedAddDelegator(
   }
 }
 
-async function getAddDelegatorParams(
+export async function getAddDelegatorParams(
```

Comment:
please add documentation for all the exported function, interfaces, constants and enums